lib/usd/utils/ForwardDeclares.h

```diff
@@ -13,7 +13,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-#pragma once
+#ifndef UTILS_FORWARDDECLARES_H
+#define UTILS_FORWARDDECLARES_H
```

Comment:
```#pragma once ``` is preferable to include guards. Also, this choice of include guard name does not conform to the provisional coding standards.